src/portal/package-lock.json

```diff
@@ -13681,7 +13681,7 @@
         },
         "serialize-error": {
             "version": "2.1.0",
-            "resolved": "http://registry.npmjs.org/serialize-error/-/serialize-error-2.1.0.tgz",
+            "resolved": "https://registry.npmjs.org/serialize-error/-/serialize-error-2.1.0.tgz",
             "integrity": "sha1-ULZ51WNc34Rme9yOWa9OW4HV9go="
         },
         "serialize-javascript": {
```

Comment:
any changes for package.json?

Reply:
If no change in package.json, why only this lock file changed?

Reply:
I have restore